Admin/PageAdmin.php

```diff
@@ -336,7 +336,7 @@ protected function configureFormFields(FormMapper $formMapper)
         if ($this->hasSubject() && !$this->getSubject()->getId()) {
             $formMapper
                 ->with('form_page.group_main_label')
-                    ->add('site', null, array('required' => true, 'read_only' => true))
+                    ->add('site', 'null', array('attr' => array('required' => 'required', 'readonly' => 'readonly')))
```

Comment:
I think `required` is a valid option, maybe only read only needs to be moved to the html attribute array?